src/bundle/Resources/public/js/CKEditor/anchor/ui/anchor-form-view.js

```diff
@@ -0,0 +1,81 @@
+import View from '@ckeditor/ckeditor5-ui/src/view';
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+import LabeledFieldView from '@ckeditor/ckeditor5-ui/src/labeledfield/labeledfieldview';
+import { createLabeledInputText } from '@ckeditor/ckeditor5-ui/src/labeledfield/utils';
+
+class IbexaLinkFormView extends View {
+    constructor(props) {
+        super(props);
+
+        this.locale = props.locale;
+
+        this.anchorInputView = this.createTextInput('Name');
+        this.saveButtonView = this.createButton('Save anchor', null, 'ck-button-save', 'save-anchor');
+        this.cancelButtonView = this.createButton('Remove anchor', null, 'ck-button-cancel', 'remove-anchor');
+
+        this.children = this.createFormChildren();
```

Comment:
Nitpick: This for me seems to be a little odd, I would move creation of `this.anchorInputView`, `this.saveButtonView` etc. to `createFormChildren` method as well. What do you think?

Reply:
Or passing `this.anchorInputView`, `this.saveButtonView` etc. as arguments to `createFormChildren`, for me, would also be more natural.

Reply:
discussed in private